acrobat/scripts/scripts.js

```diff
@@ -186,7 +186,7 @@ const locales = {
   no: { ietf: 'no-NO', tk: 'aaz7dvd.css' },
   pl: { ietf: 'pl-PL', tk: 'aaz7dvd.css' },
   pt: { ietf: 'pt-PT', tk: 'inq1xob.css' },
-  ro: { ietf: 'en-RO', tk: 'aaz7dvd.css' },
+  ro: { ietf: 'en-RO', tk: 'qxw8hzm.css' },
```

Comment:
should it be `en-RO` or `ro-RO`?

Reply:
at least for purposes of dealing with the widget and generate-cache, it should be `ro-RO` once we provide Romanian UI support

Reply:
yep it should be `ro-RO`

Reply:
Done, changed lang parameter to 'ro' https://github.com/adobecom/dc/pull/609/commits/af8b5af2468485b7d9cf0ac063972e3b0cf9fa8c
cc @Blainegunn @peller